plugin/adsk/plugin/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ target_sources(${TARGET_NAME}
         adskExportCommand.cpp
         adskListJobContextsCommand.cpp
         adskListShadingModesCommand.cpp
+        adskMaterialCommands.cpp
```

Comment:
You need to add this to the list TARGET_NAME only is UFE_FOUND is true. This will fix the `ecg-mayausd-branch-preflight-2019-no-ufe-windows` build.